client/src/pages/Home.js

```diff
@@ -1,11 +1,16 @@
 import React, { useContext } from "react";
 import { UserContext } from "../context/UserContext";
+import { NavBar } from "../components";
```

Comment:
not a fan of these relative paths, rather have it absolute from src

Reply:
changed